code/game/machinery/machinery.dm

```diff
@@ -540,6 +540,11 @@ Class Procs:
 			to_chat(user, "<span class='warning'>You momentarily forget how to use [src].</span>")
 			return 1
 
+	var/mob/cat = locate(/mob/living/simple_animal/cat) in loc
```

Comment:
Consider making it check both `loc` and the user's `loc` for added pain.

Reply:
That's pretty funny but much less likely to occur since cats have passflags for machines but they're not so undense that they just walk into the same tile as a mob.